HW5/task_1.py

```diff
@@ -0,0 +1,28 @@
+# Створити програму, яка буде очікувати від користувача введення
+# тексту і виведе інформацію по кожному надрукованому символу:
+#
+# це “число” + яке воно (парне, непарне),
+# це “буква” + яка вона (велика чи маленька),
+# це “символ”
+sample_text = input("Input some text: ")
+st_list = list(sample_text)
+
+for symbol in st_list:
```

Comment:
nit: it is not required of list using
```suggestion
for symbol in sample_text
```